oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -153,6 +153,14 @@ static int mca_spml_ucx_component_register(void)
                                      "Enable asynchronous progress thread",
                                      &mca_spml_ucx.async_progress);
 
+    mca_spml_ucx_param_register_bool("symmetric_rkey", 0,
+                                     "Enable remote keys deduplication to save memory at scale (best-effort)",
+                                     &mca_spml_ucx.symmetric_rkey);
+
+    mca_spml_ucx_param_register_int("symmetric_rkey_size", 1000,
+                                    "Size of the symmetric key store. Default (1000)",
```

Comment:
maybe keep only one parameter, "symmetric_rkey_max_count" ("size" is confusing since it can mean size of an object in bytes)
when it's set to 0, the feature is disabled. and then will not need the other boolean config to enable/disable.

Also, no need to print "Default (1000)" since oshmem_info tool already prints default values

Reply:
fixed